zipkin-storage/cassandra/src/main/java/zipkin2/storage/cassandra/SelectRemoteServiceNames.java

```diff
@@ -61,7 +61,7 @@ protected ResultSetFuture newFuture() {
     return factory.session.executeAsync(factory.preparedStatement
       .bind()
       .setString("service", service)
-      .setInt("limit_", 1000)); // no one is ever going to browse so many span names
+      .setInt("limit_", 10000));
```

Comment:
this one is not about span names

```suggestion
      .setInt("limit_", 1000)); // no one is ever going to browse so many service names
```